dracut/30ignition/module-setup.sh

```diff
@@ -41,7 +41,8 @@ install() {
         userdel \
         usermod \
         wipefs \
-        cryptsetup
+        cryptsetup \
+        setfiles
```

Comment:
The new approach against an-ever growing initramfs is to add a wrapper file instead:
https://github.com/flatcar/bootengine/blob/flatcar-master/dracut/30ignition/coreos-metadata-wrapper
and install the wrapper file as `setfiles`:
https://github.com/flatcar/bootengine/blob/flatcar-master/dracut/30ignition/module-setup.sh#L127